config/install.yaml

```diff
@@ -5075,9 +5106,6 @@ spec:
         - containerPort: 9443
           name: webhook-server
           protocol: TCP
-        - containerPort: 8080
-          name: metrics
-          protocol: TCP
```

Comment:
Why are we removing the container port for metrics?

Reply:
Tilt says it is duplicate value. I think this is because of the config(https://github.com/clastix/kamaji/blob/2b638fe09d74c1f8b088f38e4cd23c3220a2d6b4/config/manager/controller_manager_config.yaml#L6)

Reply:
Refer https://book.kubebuilder.io/component-config-tutorial/define-custom-config.html

Reply:
It's not related to the file you pointed out, rather, to these [changes](https://github.com/clastix/kamaji/pull/359/files#diff-ce0d597a9e5f20a8fe61c3ba5e6185a6b90cfef2c00bb5b5ef3e97b15b79f480).

Please, drop the removal of lines 45 to 48 from the `config/manager/manager.yaml` file since that port reference is absolutely required to support the Prometheus metrics scraping.

Reply:
Like you figured it out, it is related the CAPI issue and I will treat it as resolved.